app/src/main/res/layout/activity_powersource_layout.xml

```diff
@@ -66,25 +66,25 @@
                             android:orientation="horizontal">
```

Comment:
@marcnause The layout looks fantastic and works really well in the horizontal orientation. However, in the vertical orientation, the size of the buttons becomes too big, due to which the text in the _EditText_ cuts off. I'm attaching a screen recording for your reference.

https://github.com/fossasia/pslab-android/assets/125425881/525dd764-edc7-4d57-9a66-a7d401abff6b

Perhaps we can retain the older configuration of the vertical layout and just make changes in the horizontal one ? Let me know how this sounds to you.

Reply:
OK, one last try before I replace the vertical layout with the original one.

I made recordings with my test devices:

###  Yoga Smart Tab
https://github.com/fossasia/pslab-android/assets/11596220/508d9cca-8c38-4452-b7a1-a3d1e5823ab4

### Fairphone 4
https://github.com/fossasia/pslab-android/assets/11596220/c4419bda-e175-4dc3-b847-8b814f49846b

### Motorola Moto X Play
https://github.com/fossasia/pslab-android/assets/11596220/c95506d8-9baa-4ba1-b54f-029cbb7714ab

On the tablet the buttons of the vertical layout are huge, but I don't have any idea for a better vertical layout for large screens at the moment.

On the Moto X Play the unit of the PCS is cut off. We could create an extra horizontal layout for small screens, but the more layouts we have, the harder it is to maintain them. Any thoughts on this?